modules/nf-core/wittyer/main.nf

```diff
@@ -0,0 +1,69 @@
+process WITTYER {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "quay.io/nf-core/wittier:0.3.3"
+
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        error "WITTYER module does not support Conda. Please use Docker / Singularity / Podman instead."
+    }
+
+    input:
+    tuple val(meta), path(query_vcf), path(query_vcf_tbi), path(truth_vcf), path(truth_vcf_tbi), path(bed)
+
+    output:
+    tuple val(meta),    path("*.Stats.json")         , emit: report
+    tuple val(meta),    path("*eval.vcf.gz")         , emit: bench_vcf
+    tuple val(meta),    path("*eval.vcf.gz.tbi")     , emit: bench_vcf_tbi
+    path  "versions.yml"                             , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args  = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def regions = bed ? "--includeBed=$bed" : ""
+    """
+    script:
+    if ("$truth_vcf" == "${prefix}.vcf.gz")         error "Input and output names are the same, set prefix in module configuration to disambiguate!"
+    if ("$query_vcf" == "${prefix}.vcf.gz")         error "Input and output names are the same, set prefix in module configuration to disambiguate!"
+    if ("$truth_vcf_tbi" == "${prefix}.vcf.gz.tbi") error "Input and output names are the same, set prefix in module configuration to disambiguate!"
+    if ("$query_vcf_tbi" == "${prefix}.vcf.gz.tbi") error "Input and output names are the same, set prefix in module configuration to disambiguate!"
+    
+    mkdir bench
+
+    wittyer \\
```

Comment:
When I try it locally I always have to run
```bash
docker run wittyer
```
but I do not really know how it works with nf-core. In the `bcl2fastq` module they just call the program.

Reply:
in your local you can run as:

> docker run  quay.io/nf-core/wittyer:0.3.3.0


but in the process it should be like:

> nf-core/wittyer:0.3.3.0